client/src/components/AuthenticationPage.jsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import { Button, Modal } from 'react-materialize';
+import LoginContainer from './LoginContainer';
+import SignUpContainer from './SignUpContainer';
+
+export default function AuthenticationPage(props) {
+  const isLoggedIn = () => {
+    let isUserLoggedIn = false;
+    const token = window.localStorage.getItem('token');
+    if (token !== '' && token !== null) {
+      isUserLoggedIn = true;
+    }
+    return isUserLoggedIn;
+  };
+  if (isLoggedIn()) {
+    window.location.replace('/dashboard');
+    return null;
+  }
+
+  return (
+    <div className={'container valign-wrapper full-height'}>
```

Comment:
I agree, but it's a class name in Materialize CSS, not my own custom CSS. I have little control over it.